convert-hf-to-gguf.py

```diff
@@ -1151,11 +1151,10 @@ def set_vocab(self):
 
         # TODO: how to determine special FIM tokens automatically?
         special_vocab = gguf.SpecialVocab(self.dir_model, load_merges=False,
-                                          special_token_types = ['prefix', 'suffix', 'middle', 'fsep', 'eot'])
+                                          special_token_types = ['prefix', 'suffix', 'middle', 'eot'])
         special_vocab._set_special_token("prefix", 1)
         special_vocab._set_special_token("suffix", 3)
         special_vocab._set_special_token("middle", 2)
-        special_vocab._set_special_token("fsep",   4) # is this correct?
```

Comment:
Token id `4` in [Refact's tokenizer](https://huggingface.co/smallcloudai/Refact-1_6B-fim/blob/main/tokenizer.json) seems to be `<fim_pad>` anyway, which doesn't look like a separator, so it was not correct.

Reply:
Thanks. I've rebased the PR and resolved conflicts.